src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1840,6 +1840,79 @@ public void canCreateMultipleLoansWithServicePoints()
 
   }
 
+  @Test
+  public void canGetLoanWithPatronGroupName() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
```

Comment:
Please can at least some the throws statement be moved onto a separate line to avoid a long line here

Reply:
Yes, sure